controllers/ProfileController.js

```diff
@@ -69,6 +69,35 @@ class ProfileController {
       user: userData
     });
   }
+
+  /**
+   * @author Manzi
+   * @param {*} req
+   * @param {*} res
+   * @returns {*} Users object
+   */
+  static async getProfiles(req, res) {
+    const profiles = await User.findAll({attributes: ['firstName', 'lastName', 'image', 'bio', 'cover']});
+    return res.status(200).json({ status: 200, user: profiles });
```

Comment:
@NkFab You need to pagination because the number of users may increase to thousands and retrieving all of them at once will affect the server's performance.